src/notifications/templates/notifications/telegram/subscription_payment_failed.md

```
@@ -0,0 +1,3 @@
+{%load i18n %}⚠️ {%blocktranslate with plan=context.plan_name org=context.organization_name date=context.grace_period_end %}We couldn't collect payment for your <b>{{ plan }}</b> membership at <b>{{ org }}</b>. Resolve this by {{ date }}.{%endblocktranslate%}
```

markdownlint-cli2 (0.22.1)

[warning] First line in a file should be a top-level heading

(MD041, first-line-heading, first-line-h1)